Makefile

```diff
@@ -1,20 +1,22 @@
 
 BIN_NAME="terraform-provider-instaclustr"
 
-VERSION=v1.9.5
+VERSION=1.9.6
+INSTALL_FOLDER=$(HOME)/.terraform.d/plugins/example.com/instaclustr/instaclustr/$(VERSION)/darwin_amd64
```

Comment:
[required] does that need to be around ? example.com ? feels like some testing ?

Reply:
Yeah, I misread the docs. Have made it look more instaclustr-y, though it still just fails to resolve then points at a local folder which is super weird imo